link-dev.sh

```diff
@@ -21,6 +21,6 @@ do
   mkdir -p ${example}/node_modules/launchdarkly-vue-client-sdk/node_modules
   mkdir -p ${example}/node_modules/launchdarkly-vue-client-sdk/dist
   cp package.json ${example}/node_modules/launchdarkly-vue-client-sdk/package.json
-  cp -r node_modules/ ${example}/node_modules/launchdarkly-vue-client-sdk/node_modules/
```

Comment:
This used to exist in react-client-sdk as well. The original here happens to work on mac, but doesn't work on linux (you get dist/dist)